angular-app/src/app/app.component.ts

```diff
@@ -1,19 +1,34 @@
-import { Component, AfterViewInit, ViewChild } from '@angular/core';
-import { ToggleChatService } from './toggle-chat.service';
+import { Component, OnInit, AfterViewInit, ViewChild } from '@angular/core';
 import { MatSidenav } from '@angular/material/sidenav';
 
+import { ToggleChatService } from './toggle-chat.service';
+import { AuthService } from './auth.service';
+
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
-export class AppComponent implements AfterViewInit {
+export class AppComponent implements OnInit, AfterViewInit {
+
   @ViewChild('chat') public chat: MatSidenav;
+  public user$;
```

Comment:
My recollection of private vs public in angular is kinda fuzzy. Why does user$ need to be public?

Reply:
Private properties are unfortunately not visible to my template either; I use this in the ngIf directive in the HTML, which is why it's public.